bark/tools/dat-from-wav.py

```diff
@@ -0,0 +1,28 @@
+from scipy.io import wavfile as wavfile
+import bark, os
+
+
+def main(wav):
+	if not isinstance(wav, str):
+	    raise ValueError('A file path string is required')
+	name, ext = os.path.splitext()   # get the original file name. e.g. ab from ab.wav
+	rate, data = wavfile.read(wav)
+	# make dat file with the same name as wav file
+	datname = '{}.dat'.format(name)
+	bark.write_sampled(datname, data, rate, original_file=wav)
+
+
+def _run():
+    ''' Function for getting commandline args.'''
+    import argparse
+
+    p = argparse.ArgumentParser(description='''
+    converts wav file to bark format
+    ''')
+    p.add_argument('wav', help='path to wav file')
```

Comment:
Some additional optional command-line arguments might be useful - take a look at bark/bark/tools/lbl.py, particularly the `-o` and `-a` arguments.

Reply:
What is the correct of way of using -o and -a arguments? What does it do?
I am trying : 
python datfromwav.py abc.wav -a name=bark

Where does name=bark go?

Reply:
Hi Graham,
Particularly in -o argument, is safe to assume that user will always provide the pathname as an argument? Or do I need to check for nonetype path in argument?